### ISSUE TITLE
Update documentation to point users to jaxb-ri cli tools

### ISSUE THREAD

**Issue by ItaloBorrelli** [eclipse-ee4j/jaxb-ri#1687]
I spent far more time than I should have trying to generate java classes for an xsd schema. The mention of:

Download Jakarta XML Binding Release
The latest stable release of Eclipse Implementation of JAXB is available for [download](https://repo1.maven.org/maven2/com/sun/xml/bind/jaxb-ri/4.0.1/jaxb-ri-4.0.1.zip) as well as for consumption through maven.

is ok, but it's also inconspicuous and it took me a very long time to realize that download was a link because the text looks almost black on a white background.
![image](https://user-images.githubusercontent.com/44072946/213782546-e9e8f946-c308-4a20-8961-f2c474000819.png)

In fact, I ended up creating a new project that just ran `Driver.run(args, new PrintStream(System.out), new PrintStream(System.out));` to generate the xjc with given arguments from cli. It would save anyone a lot of time perhaps even just to have a little download icon so that someone knows they can download it there, because there's nowhere else that a user could easily find it, because it's not clear that jaxb-ri is one of your artifacts since it's not mentioned anywhere else. Even on the maven repo 

And perhaps there can be some cli usage documentation? I only saw a brief mention for schemagen cli on the user guides.

**Comment by lukasj** [comment]
re download link - there are at least 3 pointers to download - on big orange button top right, one menu item in the menu on the right hand side of the web and the link in the text you are pointing to

> And perhaps there can be some cli usage documentation?

https://eclipse-ee4j.github.io/jaxb-ri/4.0.1/docs/ch04.html is just about that it even explicitely shows:
`xjc [OPTION]... <schema file/URL/dir/jar> [-b <binding>...]` together with the output of the `xjc -help`

**Comment by ItaloBorrelli** [comment]
Ah thanks. All fair points. It's been almost half a year since I looked at this and it recently came up again and was pointed to me that there was a download for a cli tool that I didn't know about. I think there's a few factors that affected what I as a user see. First, The orange download link I would assume might be for downloading eclipse itself. Also, the side and top menus are static, so as I read the content on the page, those are out of view. Also the download link is a different colour from "This project is part of [Eclipse Implementation of JAXB](https://projects.eclipse.org/projects/ee4j.jaxb-impl)" and all the documentation links. Now that these links have been pointed out to me I can see them and I know they all download the same content, but for a first time visitor I think that it is less obvious than one might assume. I feel silly now that I know all this, but I also understand why I missed it, and I hope you might understand that perspective on it.

As for the cli documentation, I do remember now finding that when I had worked with xjc previously, I just didn't actually know how or where to download the cli tool project.

Thanks for your time though
